### PR TITLE
Change handle to email prefix

### PR DESCRIPTION
**Pull request by gjacks13**
Modify Login Plugin to set user handle to email prefix PR, for new user logins

### CODE REVIEW

**Review by dynamod92 — APPROVED** [review]
👍

File: qa-open-login.php [hunk 1]
```diff
@@ -99,18 +99,21 @@ function check_login()
 			if (!isset($topath)) {
 				$topath = ''; // redirect to front page
 			}
-
+			
+			// try to login
+			$hybridauth = new Hybrid_Auth($config);
```
Comment (jonhester) [inline qa-open-login.php]: Any reason to move this out of the try block?

Comment (celioreyes19) [reply qa-open-login.php]: My first login hit the `catch` block and it freaked out b/c the `$adapter` wasn't defined

Comment (jonhester) [reply qa-open-login.php]: But won't you still get an exception?

Comment (celioreyes19) [reply qa-open-login.php]: No, the `catch` just redirects you back to the homepage

Comment (celioreyes19) [reply qa-open-login.php]: https://github.com/RedVentures/q2a-open-login/blob/changeHandleValue/qa-open-login.php#L129-L142

**Review by jonhester — APPROVED** [review]

**Pull request merged**